data/huion-wh1409.tablet

```diff
@@ -0,0 +1,40 @@
+# HUION
+# WH1409
+#
+# sysinfo.DFvHUfcRjW.tar.gz
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/385
+#
+# Button Map:
+# (A=1, B=2, C=3, ...)
+#
+#         *----------------------------*
+#    A B  |                            |
+#    c D  |                            |
+#         |                            |
+#    E F  |           TABLET           |
+#    G H  |                            |
+#         |                            |
+#    I J  |                            |
+#    K L  |                            |
+#         *----------------------------*
+
+[Device]
+Name=TABLET Pen Tablet
```

Comment:
```suggestion
Name=Huion WH1409
```
fwiw, the purpose of the `Name=` field is to give us a nice human-readable name that's better than the garbage the firmware gives us.